doc/user/content/sql/create-cluster-replica.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "CREATE CLUSTER REPLICA"
+description: "`CREATE CLUSTER REPLICA` provisions physical resources to perform computations."
+menu:
+  main:
+    parent: commands
+---
+
+`CREATE CLUSTER REPLICA` provisions physical resources to perform computations.
+
+## Conceptual framework
+
+Where [cluster replicas](/overview/key-concepts#clusters) represent the logical
+set of dataflows you want to maintain, cluster replicas are their physical
+counterparts. Cluster replicas are where Materialize actually creates and
+maintains dataflows.
+
+Each cluster replica is essentially a clone, constructing the same dataflows.
+Each cluster replica receives a copy of all data that comes in from sources its
+dataflows use, and uses the data to performs identical computations. This design
+provides Materialize with active replication, and so long as one replica is
+still reachable, the cluster continues making progress.
+
+This also means that all of a cluster's dataflows contend for the same resources
+on each replica. This might mean, for instance, that instead of placing many
+complex materialized views on the same cluster, you choose some other
+distribution, or you replace all replicas in a cluster with more powerful
+machines.
+
+## Syntax
+
+{{< diagram "create-cluster-replica.svg" >}}
+
+### `replica_option`
+
+{{< diagram "replica-option.svg" >}}
+
+Field | Use
+------|-----
+_cluster_name_ | The cluster whose resources you want to create an additional computation of.
+_replica_name_ | A name for this replica.
+_replica_option_ | This replica's specified [options](#replica_option).
+_size_ | A "size" (i.e. number of timely workers) for a managed replica––valid values are powers of 2.
```

Comment:
I'm not sure we want to commit that the sizes are number of Timely works and powers of two. For our cloud offering, we certainly wouldn't use the builtin default values but rather pass a custom `--cluster-replica-sizes` parameter. In the past, we had the idea that the sizes would be something like t-shirt sizes, but let's sync with Cloud to get an agreement on this.

Reply:
Cloud already has a take! The sizes are `xsmall`, `small`, `medium`, `large`, `xlarge`, `2xlarge`, `3xlarge`, `4xlarge`, `5xlarge`, and `6xlarge`.

https://github.com/MaterializeInc/cloud/blob/4077c0365c33798f993940589e4843a0d1920c99/Pulumi.production.yaml#L93-L102